services/brig/src/Brig/App.hs

```diff
@@ -42,6 +42,7 @@ module Brig.App
     templateBranding,
     requestId,
     httpManager,
+    http2Manager,
```

Comment:
do you want to keep using both?  in other words: is http2-manager strictly better than http-manager, or what are the use cases for `httpManager`?

Reply:
We need both unfortunately, we cannot assume everyone outside of our internal components runs http2 compatible servers.